build/wd_cc_benchmark.bzl

```diff
@@ -23,6 +23,15 @@ def wd_cc_benchmark(
         ],
         # Only run benchmarks when explicitly requested, at least until we have some more of them
         # and can define a benchmark suite.
-        tags = ["manual", "benchmark"],
+        tags = ["off-by-default", "benchmark"],
```

Comment:
I found this more useful than manual, since there's no other way to run manual targets except directly enumerating them. You are forced to run bazel query as a separate step. fyi @fhanau

Reply:
👍 I agree, using `off-by-default` sounds like the right approach.